proxy-node-translator/src/test/java/uk/gov/ida/notification/translator/HubResponseTranslatorTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.ida.notification.translator;
+
+import io.dropwizard.testing.junit.ResourceTestRule;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.ida.common.ExceptionType;
+import uk.gov.ida.exceptions.ApplicationException;
+import uk.gov.ida.notification.contracts.HubResponseTranslatorRequest;
+import uk.gov.ida.notification.contracts.verifyserviceprovider.TranslatedHubResponseBuilder;
+import uk.gov.ida.notification.shared.Urls;
+import uk.gov.ida.notification.shared.proxy.VerifyServiceProviderProxy;
+import uk.gov.ida.notification.translator.resources.HubResponseTranslatorResource;
+import uk.gov.ida.notification.translator.saml.EidasResponseGenerator;
+import uk.gov.ida.saml.core.test.builders.ResponseBuilder;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.STUB_COUNTRY_PUBLIC_PRIMARY_CERT;
+
+@RunWith(MockitoJUnitRunner.class)
+public class HubResponseTranslatorTest {
```

Comment:
Just spotted this now - can we rename this class to `HubResponseTranslatorResourceTest` - as `HubResponseTranslatorResource` is the class we're testing here.
There is also a class called `HubResponseTranslator` which doesn't have any tests actually - did you think about writing some tests for it as well?

Reply:
Ok will do.